aperturedb/sqlc/migrations/000003_lnc-sessions.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS sessions (
+    id INTEGER PRIMARY KEY,
+
+    passphrase_entropy BLOB NOT NULL UNIQUE,
+    remote_static_pub_key BLOB UNIQUE,
+    local_static_priv_key BLOB NOT NULL,
+    mailbox_addr TEXT NOT NULL,
+
+    created_at TIMESTAMP NOT NULL,
+    expiry TIMESTAMP WITHOUT TIME ZONE
```

Comment:
Any reason why one needs the timezone info while the other doesn't?

Reply:
not really, in postgres/sqlite `TIMESTAMP` is the same as `TIMESTAMP WITHOUT TIME ZONE`. I will remove it to be consistent with the shorter version `TIMESTAMP`